<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPECheckSum</class>
 <widget class="QDialog" name="DialogPECheckSum">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>107</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>107</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>107</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>CheckSum</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxReadOnly">
       <property name="text">
        <string>Read only</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEditX" name="lineEditCheckSum">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonCalculate">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonOK">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonCancel">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonApply">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPushButtonX</class>
   <extends>QPushButton</extends>
   <header>qpushbuttonx.h</header>
  </customwidget>
  <customwidget>
   <class>QLineEditX</class>
   <extends>QLineEdit</extends>
   <header>qlineeditx.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
